### DISCUSSION TITLE
versione 1.5.8 idee.

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#637] [Idee e proposte]
Buonasera le presento delle mie idee per la prossima versione:

1.  L'aggiunta da parte della segreteria di aggiungere o modificare voti non agli alunni ma creare una valutazione  e aggiungerla alla lista dei docenti (per esempio aggiungere il voto impreparato)
2. Modifica del sistema di valutazione (es. A B ecc.)
3. la possibilità di stampare gli elenchi degli alunni, docenti ecc.
4. la possibilità di aggiungere la scuola media
5. il voto + e -
6. la nota generica ( quando si inserisce una nota si sceglie tra disciplinare o generica
7. l'orario
8. mettere i compiti insieme alla firma
9. i voti più grandi con la data sopra (tipo argo)  
10. la possibilità di far vedere ai genitori il fuori classe
11. la possibilità di fare relazioni agli alunni che si comportano male
12. la funzione importa voti agli scrutini
13. la funzione (nell' app) che ogni volta che un docente mette un voto, nota ecc. che arriva una notifica all' alunno/ genitore con scritto ci sono novità (evento) per (nome alunno)

**Comment by Davide031** [comment]
ma anche mettere la presa visione ad una nota

**Comment by trinko** [comment]
6) se la nota non è disciplinare, è indicata come ANNOTAZIONE

Per il resto: tutte belle idee. 
Chiunque si senta libero di realizzarle, io poi le inserisco nel codice dell'applicazione: così funziona l'OPEN SOURCE.

**Reply by Davide031** [reply]
Avevo anche un altra idea che quando metto una nota devo indicare l'orario di quando è successo il fatto.
